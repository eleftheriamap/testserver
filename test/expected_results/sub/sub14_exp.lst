
test/exp/test/gen/sub/sub14_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d16b9ab5 	sub	x21, x21, #0xae6, lsl #12
   4:	d15a61ce 	sub	x14, x14, #0x698, lsl #12
   8:	4b8e7aba 	sub	w26, w21, w14, asr #30
   c:	8a000000 	and	x0, x0, x0
