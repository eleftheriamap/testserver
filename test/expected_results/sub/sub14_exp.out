Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = ffffffffff968000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = ffffffffff51a000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 00000000ff51a001
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd16b9ab5
0x00000004 : 0xd15a61ce
0x00000008 : 0x4b8e7aba
0x0000000c : 0x8a000000
